src/QMCWaveFunctions/TrialWaveFunction.cpp

```diff
@@ -667,6 +651,34 @@ void TrialWaveFunction::evaluateDerivatives(ParticleSet& P,
   }
 }
 
+void TrialWaveFunction::evaluateDerivativesWF(ParticleSet& P,
+    const opt_variables_type& optvars,
+    std::vector<ValueType>& dlogpsi,
+    bool project)
```

Comment:
You are the only one using evaluateDerivativesWF. Get ride of project.

Reply:
This argument has been removed.